apps/linode-marketplace-beef/provision.yml

```diff
@@ -0,0 +1,23 @@
+# peppermint
+- name: setting up environment 
+  hosts: localhost
+  connection: local
+  any_errors_fatal: true
+  user: root
+  vars_files:
+    - group_vars/linode/vars
+  tasks:
+  # generate creds
+    - name: generating sudo user and password 
+      import_tasks: ../linode_helpers/roles/sudouser/tasks/main.yml
+
+    - name: generating beef user password
+      set_fact:
+        BEEFPASSWORD: "{{ lookup('password', '/dev/null length=45 chars=ascii_letters,digits') }}"
+
+    - name: writing beef to vars
+      lineinfile: 
+        insertafter: EOF
+        path: group_vars/linode/vars
+        line: | 
+          beefpassword: {{ BEEFPASSWORD }}
```

Comment:
update to:
`beefpassword: {{ lookup('password', '/dev/null length=45 chars=ascii_letters,digits') }}`